lib/Yam/ControlCenterPage.pm

```diff
@@ -0,0 +1,368 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Control Center
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::ControlCenterPage;
+use parent 'Yam::PageBase';
+use strict;
+use warnings;
+use testapi;    # needed because of bsc1206966
+
+sub init {
+    my $self = shift;
+    $self->{lbl_yast_cc} = $self->{app}->label({label => 'YaST Control Center'});
+    $self->{slb_groups} = $self->{app}->selectionbox({id => 'groups'});
+    $self->{slb_progs} = $self->{app}->selectionbox({id => 'progs'});
+    $self->{btn_help} = $self->{app}->button({id => 'help'});
+    $self->{btn_run} = $self->{app}->button({id => 'run'});
+    $self->{btn_quit} = $self->{app}->button({id => 'quit'});
+    return $self;
+}
+
+sub get_control_center_page {
```

Comment:
I missed something important on the first draft, I can see now that this method is actually sort of private (we don't use it from the test in general but can be used as public as a workaround).
I actually would recommend to name is as `_get_control_center_page`, include in private, and then the most important, we add it in front of each other action we do, example:

```
sub _select_systemd_journal {
    my ($self) = @_;
    get_control_center_page()->{slb_progs}->select('Systemd Journal');
}
```